files/default/neutron-ha-tool.py

```diff
@@ -322,7 +329,7 @@ def l3_agent_rebalance(qclient, noop=False, wait_for_ports=False):
             if migrate_router_safely(qclient, noop, router_id,
                                      l3_agent_dict[hgh_agent_id],
                                      l3_agent_dict[low_agent_id],
-                                     wait_for_ports):
+                                     wait_for_ports, False):
```

Comment:
Think I agree with @aplanas - again here we have the [Too Many Parameters code smell](https://sourcemaking.com/refactoring/smells/long-parameter-list) for different reasons, but this compounds the readability issue caused by passing a raw boolean as an argument.  Would be much more readable to use named parameters.